kolibri/plugins/device_management/assets/src/views/ManageContentPage/SelectTransferSourceModal/SelectImportSourceModal.vue

```diff
@@ -41,26 +47,33 @@
   import KRadioButton from 'kolibri.coreVue.components.KRadioButton';
   import { RemoteChannelResource } from 'kolibri.resources';
   import KModal from 'kolibri.coreVue.components.KModal';
+  import UiAlert from 'keen-ui/src/UiAlert';
   import { ContentSources } from '../../../constants';
 
   export default {
     name: 'SelectImportSourceModal',
     components: {
       KRadioButton,
       KModal,
+      UiAlert,
     },
     data() {
       return {
         source: ContentSources.KOLIBRI_STUDIO,
         formIsDisabled: true,
         kolibriStudioIsOffline: false,
         ContentSources,
+        showNotifications: false,
       };
     },
     computed: {
       ...mapGetters('manageContent/wizard', ['isImportingMore']),
     },
     created() {
+      // Only show notification if things take too long
+      setTimeout(() => {
```

Comment:
Can probably remove the `setTimeout` and `showNotification` flag, as it doesn't seem to benefit the slower case, and can cause a weird popin/out effect in the faster case.

![notification](https://user-images.githubusercontent.com/1680573/45720788-8d3cb800-bb73-11e8-904a-f52c5a7f06d7.gif)

Reply:
I tried it without the setTimeout and it seemed to be a slightly less jarring experience.